lti_consumer/templates/html/student.html

```diff
@@ -1,18 +1,23 @@
-<h2 class="problem-header">
-    ## Translators:  "External resource" means that this learning module is hosted on a platform external to the edX LMS
-    ${display_name} (External resource)
-</h2>
+% if display_header:
+    <h2 class="problem-header">
+        ## Translators:  "External resource" means that this learning module is hosted on a platform external to the edX LMS
+        ${display_name} (External resource)
```

Comment:
I'm getting confused by this code.  Does translation ever happen if no call to _() is made?  Shouldn't this be something like?:
```
${_('{display_name} (External resource)').format(display_name=display_name)}
```
Assuming this is correct, there is missing translation code throughout.  Is something different done with xblocks?